internal/store/postgres/scheduler/job_repository.go

```diff
@@ -243,6 +243,43 @@ func fromStorageWindow(raw []byte, jobVersion int) (window.Config, error) {
 	return window.NewCustomConfig(w), nil
 }
 
+type change struct {
```

Comment:
why we couldn't create a repository for changlog & use it in both job & resource repository.